src/main/java/hudson/tasks/junit/History.java

```diff
@@ -248,7 +248,13 @@ private void createLinearTrend(
         if (history.size() < 3) {
             return;
         }
-        double[] cs = LeastSquares.calcCoefficients(lrX, lrY);
+
+        SimpleRegression sr = new SimpleRegression(true);
+        for (int i = 0; i < lrX.length; i++) {
+            sr.addData(lrX[i], lrY[i]);
```

Comment:
A bit awkward, but the `SimpleRegression` methods that accept arrays of data expect `double[n][2]` instead of `double[2][n]`, so I did not see a simpler approach.